java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/protocol/WorkspaceServiceImpl.java

```diff
@@ -397,6 +398,47 @@ public CompletableFuture<Object> executeCommand(ExecuteCommandParams params) {
                     return future;
                 });
             }
+            case Server.JAVA_RESOLVE_STACKTRACE_LOCATION: {
+                CompletableFuture<Object> future = new CompletableFuture<>();
+                try {
+                    String uri = ((JsonPrimitive) params.getArguments().get(0)).getAsString();
+                    String methodName = ((JsonPrimitive) params.getArguments().get(1)).getAsString();
+                    String fileName = ((JsonPrimitive) params.getArguments().get(2)).getAsString();
+
+                    FileObject fo = Utils.fromUri(uri);
+                    ClassPath classPath = ClassPathSupport.createProxyClassPath(new ClassPath[] {
```

Comment:
Guard against `fo == null`, will NPE in `getClassPath`

Reply:
Done.